server/startup/initialData.js

```diff
@@ -28,7 +28,7 @@ Meteor.startup(async function() {
 			type: 'bot',
 		});
 
-		addUserRoles('rocket.cat', 'bot');
+		addUserRolesSync('rocket.cat', 'bot');
```

Comment:
this function is already `async`, so:
```suggestion
		await addUserRoles('rocket.cat', 'bot');
```